jdi-light-bootstrap-tests/src/main/java/io/github/com/pages/BootstrapPage.java

```diff
@@ -136,6 +135,9 @@
 import io.github.com.sections.spinner.SpinnerAlignment;
 import io.github.com.sections.spinner.SpinnerSize;
 
+//--- navbar package
```

Comment:
to remove

Reply:
done:
import io.github.com.sections.spinner.SpinnerAlignment;
import io.github.com.sections.spinner.SpinnerSize;

public class BootstrapPage extends WebPage {
    //region Simple Elements
    public static UIElement inputGroupWrap;
    public static UIElement inputGroupNowrap;
    public static Button redButton;
    public static Button doubleButton;
    public static Button disabledButton;